packages/twenty-server/src/engine/core-modules/feature-flag/constants/public-feature-flag.const.ts

```diff
@@ -0,0 +1,14 @@
+import { FeatureFlagKey } from 'src/engine/core-modules/feature-flag/enums/feature-flag-key.enum';
+
+export type PublicFeatureFlag = Extract<
+  FeatureFlagKey,
+  | 'IS_ANALYTICS_V2_ENABLED'
+  | 'IS_ADVANCED_FILTERS_ENABLED'
+  | 'IS_COMMAND_MENU_V2_ENABLED'
+>;
+
+export const PUBLIC_FEATURE_FLAGS: PublicFeatureFlag[] = [
```

Comment:
Ask Thomas for a nice empty state since there might not be anything available yet... The first thing we will add is probably workflows